### PR TITLE
Fix for #8

### PR DESCRIPTION
**Pull request by twomice**

### CODE REVIEW

File: CRM/Utils/Normalize.php [hunk 1]
```diff
@@ -222,6 +222,10 @@ function normalize_address(&$address) {
       'FR' => '/^(\d{5})$/i',
       'NL' => '/^(\d{4})\s*([a-z]{2})$/i',
     );
+    // These will be all-capped
+    $directionals = array(
+      'n', 's', 'e', 'w', 'ne', 'nw', 'se', 'sw'
```
Comment (nganivet) [inline CRM/Utils/Normalize.php]: You do not need the 1 letter directions here since they will be capitalized with ucfirst()

**Comment by twomice** [comment]
Thanks, Nicolas!

**Comment by twomice** [comment]
@nganivet Here's that little extra fix for the single-letter directionals.

**Comment by twomice** [comment]
Thanks, Nicolas!

**Pull request merged**